.github/workflows/scheduled-image-scan.yml

```diff
@@ -45,7 +45,8 @@ jobs:
 
       - name: Upload sarif file to Github Code Scanning
         if: always()
-        uses: github/codeql-action/upload-sarif@v2
+        continue-on-error: true #avoid fail the pipeline if the SARIF upload fails.
```

Comment:
What are some example upload failures?

Reply:
Hi @terrytangyuan, here ir is: https://github.com/kserve/kserve/actions/runs/8405425651